lambo

```diff
@@ -226,6 +229,9 @@ while [[ $# -gt 0 ]]; do
         -n|--node)
             NODE=true
             ;;
+        -m|--mysql)
```

Comment:
Aah, sorry, don't know how I missed that. Maybe `-c` or `--create` although that is pretty vague.